src/Chainweb/Pact/Validations.hs

```diff
@@ -64,7 +65,7 @@ assertLocalMetadata
     :: P.Command (P.Payload P.PublicMeta c)
     -> TxContext
     -> Maybe LocalSignatureVerification
-    -> PactServiceM tbl (Either MetadataValidationFailure ())
+    -> PactServiceM tbl (Either Text ())
```

Comment:
Why aren't we classifying this text value anymore?

Reply:
`MetadataValidationFailure` was a newtype over `Text` originally, but I've lifted it into the ADT `LocalResult` here, which gets built directly in `execLocal` in `PactService.hs` to avoid committing this code to just being "local-specific", if that makes sense. It's a general series of validations, so it should deal with generally producing messages that other places in the code can use, and we can separate concerns a bit by lifting the result into whatever datatype we care to at the appropriate place.